templates/common/infra/bicep/abbreviations.json

```diff
@@ -131,5 +131,6 @@
     "webSitesAppService": "app-",
     "webSitesAppServiceEnvironment": "ase-",
     "webSitesFunctions": "func-",
-    "webStaticSites": "stapp-"
+    "webStaticSites": "stapp-",
+    "apimService": "apim-"
```

Comment:
It's already there as apiManagementService

Reply:
Ok I will remove the "apimService"

Reply:
fixed